templates/html/.partials/schema-prop.html

```diff
@@ -89,14 +89,6 @@
             {% endfor %}
           </div>
         {% endif %}
-        {% if prop.examples() %}
-          <div class="text-xs">
-            Examples:
-            {% for value in prop.examples() %}
-              <span class="border text-orange rounded ml-1 py-0 px-2">{{value | dump(2) | safe }}</span>
-            {% endfor %}
-          </div>
-        {% endif %}
```

Comment:
This doesn't remove the examples **only** for the payload block but for examples of inner properties too.

Reply:
Ah, I must have missed that as I do not have examples for inner properties. I'll revert the commit related to this -- also taking your advice into account.